cisco_aci/datadog_checks/cisco_aci/tags.py

```diff
@@ -9,66 +9,113 @@
 from . import helpers
 from . import exceptions
 
+# TODO create constrant for json attr that are used often
```

Comment:
I think most of the commonly used ones are things like "name" and "dn". Constants are good but unclear that they'd be more helpful here.